VideoLocker/src/main/java/org/edx/mobile/http/OfflineRequestInterceptor.java

```diff
@@ -27,16 +31,18 @@ public OfflineRequestInterceptor(Context context, int maxStaleTime){
     }
 
     @Override
-    public void intercept(RequestInterceptor.RequestFacade request) {
+    public Response intercept(Chain chain) throws IOException {
+        final Request.Builder requestBuilder = chain.request().newBuilder();
         if (isNetworkConnected()) {
             //TODO? should we specify caching here?
 //            int maxAge = 60; // read from cache for 1 minute
-//            request.addHeader("Cache-Control", "public, max-age=" + maxAge);
-            request.addHeader("Cache-Control", "public");
+//            builder.header(("Cache-Control", "public, max-age=" + maxAge);
```

Comment:
This is part of the suggested implementation of the TODO, and we should first [finalize][1] that we're not going to to it before deleting it. The proposed implementation should probably be updated, but that's out of scope here. I'll update the variable to be contextually correct though.

[1]: https://github.com/edx/edx-app-android/pull/799#discussion_r77185423